applications/launchpad/gui-react/src/components/NodeBox/index.tsx

```diff
@@ -40,6 +41,12 @@ const NodeBox = ({
   children,
   testId = 'node-box-cmp',
 }: NodeBoxProps) => {
+  const setSvgGradient =
+    tag?.content === 'Ready to go' ||
+    tag?.content === 'Ready to set' ||
```

Comment:
`NodeBox` is a generic component, it should not try to interpret the `tag.content` itself. I believe that it comes from Mining, so I would suggest to make decision about this specific styling in `MiningBox` container. For instance, there are two configs:
- `const defaultConfig = { .... } ` around line 131
- `const defaultStates = { .... } ` around line 150

You can add another optional prop for the tag color.

And btw, two notes, but if you apply above, then these won't be applicable:
a) prefix `set` in `setSvgGradient` suggest an _action_. In this case just `svgGradient` would be good enough
b) It would be better to compare against locale rather than static text. 

```
// So it should be:
tag?.content === t.common.readyToGo
// ..instead of:
tag?.content === 'Ready to go'
```

Reply:
Fixed (eventually!)